modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPaste.ts

```diff
@@ -0,0 +1,143 @@
+import { RealClipboard, RealMouse } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyApis, TinyAssertions, TinyHooks, TinyUi } from '@ephox/mcagar';
```

Comment:
```suggestion
import { TinyApis, TinyAssertions, TinyHooks, TinyUi } from '@ephox/wrap-mcagar';
```

TinyMCE tests should always use the `wrap-mcagar` library instead of `mcagar` directly otherwise it won't use local changes when testing. We have a Jira in the backlog to add a linter rule for this IIRC, but it's not been scheduled yet.

Reply:
done